### DISCUSSION TITLE
How to report a staff member?

### DISCUSSION THREAD

**Discussion by cameron008** [chromiecraft/chromiecraft#2627] [General]
Hello, I would like to report a staff member for breaking rules. I tried to make a ticket on Discord, but the staff member i wanted to report answered it and is being rude. The reason i am reporting him is not because he is rude, its a whole different matter.

**Comment by 55Honey** [comment]
Hello cameron008,
The way to reach out to the Head of Moderation or Admins is in a ticket on discord.
You already made a ticket about it. And you got an answer in it before posting this as well. 
Please check your discord.

**Reply by cameron008** [reply]
Oh, i did not notice the reply in my ticket. Thank you so much, everything is being handled now. <3
